src/BannerContent.tsx

```diff
@@ -70,21 +61,18 @@ export default class BannerContent extends React.Component<Props> {
     </button>
   )
 
-  templateLink = (style: React.CSSProperties, link?: Props['link']) => {
-    if (link) {
-      const { url: href, target, rel, msg } = link;
-      return (
-        <a className='cookie-link' {...{ href, target, style, rel }}>
-          {msg || 'Learn more'}
-        </a>
-      );
-    }
-  }
+  templateLink = (link: JSX.Element, style: React.CSSProperties) => (
+    React.cloneElement(link, link.props.style ? undefined : { style })
```

Comment:
do we still need a clone element? style could be passed from outside together with the element itself instead?

Reply:
I mean, does it still make sense to have a `style` for `link` if we receive a completely custom element from the caller?